README_2nd_integration.md

```diff
@@ -1,40 +1,31 @@
 # Tool name: Ontology Type Inference (OTI)
 
----
+## General description
 
-# General description
+OTI supports other code analysis tools by propagating the ground truths about ontic types in the corpus. For example, ontic types help to distinguish two functions with the same Java types and control flow graphs. OTI takes a minimal set of ground truths from other tools and then propagates them properly in the corpus by type inference based on ontology type rules.
```

Comment:
For version control it is better to have line breaks.
If you edit a character in the line, the diff will need to store the whole line, which makes it hard to see the change.
With many separate lines, you only have to look at a short line to see the changes.